File: typescript/src/main/vsdk/toolkit/io/context/TransformContext.ts
import { Matrix4x4d } from "../../common/linealAlgebra/Matrix4x4d";

export class TransformContext {
  public transformMatrix: Matrix4x4d;
  public scaleFactor: number;

  public constructor() {
    this.transformMatrix = new Matrix4x4d();
    this.scaleFactor = 0.0;
  }
}

File: java/base/src/main/java/vsdk/toolkit/io/context/TransformContext.java
package vsdk.toolkit.io.context;

import vsdk.toolkit.common.linealAlgebra.Matrix4x4d;

public class TransformContext {
    public Matrix4x4d transformMatrix;
    public double scaleFactor;

    public TransformContext() {
        transformMatrix = new Matrix4x4d();
        scaleFactor = 0.0;
    }
}
